docs/blog/2025-01-21-project-references.md

```diff
@@ -0,0 +1,177 @@
+---
+title: What Are TypeScript Project References?
```

Comment:
I wonder if a title like "A Beginner's Guide to TypeScript Project References" or "Everything You Need to Know About TypeScript Project References".

Something that has a bit more SEO juice